.codeclimate.yml

```diff
@@ -5,9 +5,17 @@ engines:
     enabled: true
   pylint:
     enabled: true
-exclude_paths:
-- migrations/
-- tests/
+  duplication:
+    enabled: true
+    config:
+      languages:
+        python:
+          mass_threshold: 100 
+          python_version: 3
```

Comment:
will this work with python 2.7?

Reply:
@koradon Yes, it's only suggestion to codeclimate parser how to score syntax, shouldn't make much difference since our code works both ways. Our main effort goes into python3-only future, so I've made it default.

Reply:
just checking :)